lib/puppet/defaults.rb

```diff
@@ -1758,8 +1758,16 @@ def self.initialize_default_settings!(settings)
         for the node stored in puppetdb are current. However, this will double the fact
         submission load on puppetdb, so it is disabled by default.",
     },
+    :publicdir => {
+      :default  => nil,
+      :type     => :directory,
+      :mode     => "0755",
+      :desc     => "The directory where Puppet public files are stored. Generally,
```

Comment:
Saw the comment from Josh here, not sure if it still applies.

Reply:
Didn't experience any spurious service restarts myself but I wasn't really looking/searching for them either. I'm fine with removing the "Generally..." part.